client/js/screens/ServiceLoading/ServiceLoadingContainer.js

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import {getOnboarding} from '../../config/models';
+import ServiceLoading from './ServiceLoading';
+
+class ServiceLoadingContainer extends Component {
+  componentDidMount() {
+    this._bootstrapAsync();
+  }
+
+  _bootstrapAsync = async () => {
+    const isOnboarding = await getOnboarding();
```

Comment:
this getOnboarding method should return a boolean. then you can do isOnboarding ? ' Signin' : 'Onboarding'

Reply:
Done